docs/perf_analyzer.md

```diff
@@ -453,6 +453,28 @@ By default perf_analyzer uses HTTP to communicate with Triton. The GRPC
 protocol can be specificed with the -i option. If GRPC is selected the
 --streaming option can also be specified for GRPC streaming.
 
+### SSL/TLS Support
+
+perf_analyzer allows communication accross a secured channel using both gRPC and HTTP protocols. Just setting the perf_analyzer SSL options does not ensure the secure communication. For gRPC, the Triton Server supports running with SSL/TLS enabled. For HTTP, the Triton server should be running behind `https://` proxy such as nginx. perf_analyzer can then establish a secure channel to the proxy.
+
+When running perf_analyzer with gRPC protocol, use any of the below options:
```

Comment:
also add a link to this:

gRPC documentation on how they can use SSL without error and no SSL without error

Reply:
added comment and your link 👍